src/calculator/service.ts

```diff
@@ -85,8 +85,12 @@ function removePunctuation(text: string) : string{
   return text.replace(/(~|`|!|@|#|$|%|^|&|\*|\(|\)|{|}|\[|\]|;|:|"|'|<|,|\.|>|\?|\/|\\|\||-|_|\+|=)/g,'')
 }
 
-function cleanText(text: string) : string{
-  return removePunctuation(removeMention(removeURL((text))))
+function removeHashtag(text: string) {
+  return text.replace(/(\s|^)#\w\w+\b/g, '')
```

Comment:
The regex does not match the following tweets:

>Years of research and development went into this day. Now all that’s left to do is run.#Breaking2

> Takeoff in 3… 2… 1… 🚀#Breaking2 is now #LIVE at https://nikebreaking2.twitter.com

The reason is that it doesn't detect the hashtag when it is preceded by a character.

I am looking at the official twitter library to extract hashtags but the regex they have is pretty complex, it is because hashtags can also have Arabic text 

Should we take emojis in a text for discussion in the next meeting with the team?

Reply:
Also, we could discuss with the teacher in the next meeting, what happens with tweets and emojis?!

Reply:
Here is the official twitter library if we need to use it for something https://github.com/twitter/twitter-text/tree/master/js

@german1608